packages/hadron-build/signtool/signtool.go

```diff
@@ -55,12 +59,15 @@ func main() {
 	cmd := exec.Command("node", "-e", script)
 	fmt.Println("Running command:", cmd.String())
 
-	cmd.Stdout = os.Stdout
-	cmd.Stderr = os.Stderr
+	stdoutStderr, err := cmd.CombinedOutput()
 
-	err := cmd.Run()
 	if err != nil {
 		fmt.Println("Error signing the file")
+		fmt.Printf("%s\n", stdoutStderr)
+		log.Println(err)
```

Comment:
Ideally this would be `log.fatal(err)` but then we don't see any of the logging and electron winstaller throws a huge error, none of which is helpful. Maybe it doesn't flush its subprocess's output? I'm not sure.